src/main/java/edu/ucsb/cs156/courses/entities/CoursesAdded.java

```diff
@@ -0,0 +1,37 @@
+package edu.ucsb.cs156.courses.entities;
+
+import javax.persistence.Entity;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.GeneratedValue;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Entity(name = "coursesadded")
+public class CoursesAdded {
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private long id;
+
+  @ManyToOne
+  @JoinColumn(name = "user_id")
+  private User user;
+  //private String id;
+  private String enrollCd;
+  private String psId;
```

Comment:
psId should be a Long not a String.  It will be a foreign key to the PersonalSchedule table.

Reply:
And... since you made the @ManyToOne for the User table, you should the same for the psId... 

That is, the PersonalSchedule entity should have a @OneToMany relationship to the CoursesAdded table.  

You can see an example in this article:

https://attacomsian.com/blog/spring-data-jpa-one-to-many-mapping